### ISSUE TITLE
Users can take part in tournaments

### ISSUE THREAD

**Issue by fluid-movement** [freestyle-players-association/fpa-app#12]
Users who are not organizers can be "interested" or "attending" a tournament.

We need to create an association and the actions for this.

**Comment by zzZCodeZzz** [comment]
True, should be implemented analog to the tournament_organizers relation.
